playbook/app/pb_kits/playbook/pb_fixed_confirmation_toast/_fixed_confirmation_toast.scss

```diff
@@ -7,7 +7,7 @@ $pb_pill_height: 25px;
 
 
 $confirmation_toast_colors: (
-  neutral:        $neutral,
+  neutral:        $steel_blue,
```

Comment:
Hi @gmorris1242 , based on Kristi's feedback on the runway story, lets use the $text_lt_light token here and not create a new token on this story